src/Spectre.Data/Datasets/RoiDataset.cs

```diff
@@ -0,0 +1,65 @@
+﻿/*
+ * Roi.cs
+ * Class represeting dataset with data of regions of interest.
+
+   Copyright 2017 Roman Lisak
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Spectre.Data.Datasets
+{
+    /// <summary>
+    /// Dataset containing information about a single roi.
+    /// </summary>
+    public class RoiDataset
```

Comment:
I would say, that such a set of points is a `Roi`, just like you've commented straight above.